service_invocation/csharp/http/checkout/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var baseURL = (Environment.GetEnvironmentVariable("BASE_URL") ?? "http://localhost") + ":" + (Environment.GetEnvironmentVariable("DAPR_HTTP_PORT") ?? "3500");
+
+var client = new HttpClient();
+client.DefaultRequestHeaders.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
+
+for (int i = 1; i <= 10; i++) {
+    var order = new Order(i);
+    var orderJson = JsonSerializer.Serialize<Order>(order);
+    var content = new StringContent(orderJson, Encoding.UTF8, "application/json");
+
+    // Adding app id as part of th header
+    client.DefaultRequestHeaders.Add("dapr-app-id", "order-processor");

```

Comment:
Since the client is scoped at the class level, let's just do this once and move below line 8.